src/test/java/org/zalando/nakadi/validation/JSONSchemaValidationTest.java

```diff
@@ -18,105 +17,13 @@
-                        "\"extra\": \"i should be no problem\"}");
-        final JSONObject nestedSchemaViolation = new JSONObject(
-                "{\"bar\": {\"foobar\": \"baz\"}, \"extra\": \"i should be no problem\", \"name\": \"12345\"}");
-
-        assertThat(validator.validate(validEvent), isAbsent());
-        assertThat(validator.validate(invalidEventMissingBar).get().getMessage(),
-                equalTo("#: required key [bar] not found"));
-        assertThat(validator.validate(invalidEventMissingNameField).get().getMessage(), equalTo("name is required"));
-        assertThat(validator.validate(nestedSchemaViolation).get().getMessage(),
-                equalTo("#: 3 schema violations found\n#/bar: 2 schema violations found\n#/bar: required key [foo]" +
-                        " not found\n#/bar: required key [bar] not found\n#: required key [foo] not found"));
-    }
-
-    @Test
-    public void schemaValidationShouldRespectIgnoreConfigurationMatchRegular() {
-        final EventType et = EventTypeTestBuilder.builder().name("some-event-type")
-                .schema("{\"type\": \"object\", \"properties\": {\"field-that-will-not-be-found\": " +
-                        "{\"type\": \"object\"}, \"event-type\": {\"type\": \"string\"}}, \"required\": " +
-                        "[\"field-that-will-not-be-found\", \"event-type\"]}").build();
-
-        final ValidationStrategyConfiguration vsc1 = new ValidationStrategyConfiguration();
-        vsc1.setStrategyName(EventBodyMustRespectSchema.NAME);
-        vsc1.setAdditionalConfiguration(new JSONObject(
-                "{\"overrides\": [{\"qualifier\": {\"field\": \"event-type\", \"match\" : \"D\"}, " +
-                        "\"ignoredProperties\": [\"field-that-will-not-be-found\"]}]}"));
-        et.getValidationStrategies().add(vsc1);
-
-        final EventTypeValidator validator = new EventTypeValidator(et).withConfiguration(vsc1);
-
-        final JSONObject event = new JSONObject(
-                "{\"event-type\" : \"X\", \"extra\": \"i should be no problem\", \"name\": \"12345\", " +
-                        "\"field-that-will-not-be-found\": " +
-                        "{\"val\": \"i must be present since the matcher will not succeed\"}}");
-        final JSONObject eventDelete = new JSONObject(
-                "{\"event-type\" : \"D\"}");
-        final JSONObject invalidEvent = new JSONObject(
-                "{\"event-type\" : \"X\", \"extra\": \"i should be no problem\", \"name\": \"12345\"}");
-        assertThat(validator.validate(event), isAbsent());
-        assertThat(validator.validate(eventDelete), isAbsent());
-        assertThat(validator.validate(invalidEvent).get().getMessage(), equalTo("#: required key " +
-                "[field-that-will-not-be-found] not found"));
-    }
-
-    @Test
-    public void schemaValidationShouldRespectIgnoreConfigurationMatchQualified() {
```

Comment:
This test no longer makes sense since this code has been finally deleted. It was never used in production.